example/src/lib.rs

```diff
@@ -9,7 +9,7 @@ use uuid::Uuid;
 
 #[derive(Serialize, TS)]
 #[ts(rename_all = "lowercase")]
-#[ts(export, export_to = "bindings/UserRole.ts")]
+#[ts(export, export_to = "UserRole.ts")]
 enum Role {
     User,
     #[ts(rename = "administrator")]
```

Comment:
Before, renaming the file from `Role.ts` to `UserRole.ts` required adding the `bindings/` prefix back again.  
With this change, this is no longer necessary.